### ISSUE TITLE
Love this rom

### ISSUE THREAD

**Issue by j-byrd** [TeamHorizon/vendor_xenonhd#16]
Is there any way i could possibly get 1 of you guys to port this rom to either the lg k20 plus(lgmp260) or moto g stylus (xt2043-4) ive had 1 unofficial build and your rom is absolutely amazing i would greatly appreciate it. Thank you in advance either way. Keep up this fantastic work love it.

**Comment by jeremiah40719** [comment]
I agree I have the moto g stylus sofiar xt2043 also and it would be so awesome to use this rom again on it. maybe even with viper and dolby added since the speakers are so good
